src/css/home.scss

```diff
@@ -397,22 +397,6 @@ $searchbox-height: 50px;
         white-space: nowrap;
       }
     }
-
-    .leadquote {
-      text-size: 14px;
```

Comment:
This wasn't doing anything so I removed it. If we want to keep it it should be `font-size` instead.